services/tasks-svc/internal/patient/aggregate/aggregate_test.go

```diff
@@ -15,3 +18,137 @@ func MustApplyEvent(t *testing.T, aggregate hwes.Aggregate, newEvent func() (hwe
 		t.Error(err)
 	}
 }
+
+func TestPatientAggregate_CreatePatient(t *testing.T) {
```

Comment:
@MaxSchaefer is probably very happy about these, I, for one, think these kinds of tests are a burden to maintain and don't increase code confidence very much

Reply:
Are you only referring to the aggregate tests here? Open to discuss :) (without blocking this pr)